src/components/Rating/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+
+const Rating = ({ initialRating, ratingCount, ratingEmoji }) => {
+  const [rating, setRating] = useState(initialRating);
+  const [emoji, setEmoji] = useState('👍🏻');
+  const [hover, setHover] = useState(null);
+  const thumbElements = [...Array(ratingCount || 5)];
+
+  useEffect(() => {
+    switch (ratingEmoji) {
+      case 'heart': {
+        setEmoji('❤️');
+
+        break;
+      }
+      case 'smiley': {
+        setEmoji('🙂');
+
+        break;
+      }
+      case 'star': {
+        setEmoji('⭐');
+
+        break;
+      }
+      default: {
+        setEmoji('👍🏻');
+      }
+    }
+  }, []);
+
+  return (
+    <div className="rating">
+      { thumbElements.map((thumb, index) => {
+        const ratingValue = index + 1;
+
+        return (
+          <label key={ index }>
+            <input
+              type="radio"
+              className="rating__radio"
+              value={ ratingValue }
+              onClick={ () => setRating(ratingValue) }
+            />
+            <span
+              role="img"
+              aria-label="thumbs"
+              className={
+                ratingValue <= (hover || rating)
+                  ? 'rating__thumb rating__thumb--selected'
```

Comment:
How about renaming the class name to be more generic because the icon can change to star/heart/smile? 🤔 what do you think?

Reply:
Yes exactly, It was only thumb rating before but the design changed recently. So I changed the other places but somehow missed this one. 😬 Fixed in [6feba1f](https://github.com/rafayet-monon/surveyor/pull/20/commits/6feba1f11e1136edeb3a544a514a0ee35dc1ebc7)